.github/workflows/windows.yml

```diff
@@ -71,12 +71,15 @@ jobs:
           echo "https://www.ibm.com/docs/en/icos/22.1.1?topic=2211-silent-installation-cplex-optimization-studio"
           curl.exe --output cplex.exe $ENV:CPLEX_URL
 
+          # This directory only already exists for repos called "downward".
+          mkdir D:\a\downward
+
           echo "Install CPLEX"
-          Start-Process -FilePath .\cplex.exe -ArgumentList "-f", "D:\a\downward\downward\.github\workflows\misc\cplex2211_windows_installer.properties" -PassThru | Wait-Process
+          Start-Process -FilePath .\cplex.exe -ArgumentList "-f", "$ENV:GITHUB_WORKSPACE\.github\workflows\misc\cplex2211_windows_installer.properties" -PassThru | Wait-Process
           del .\cplex.exe
 
           echo "Copy the relevant directory to a location which is not magically protected against cmake"
-          Xcopy /E /I ..\cplex_temp\cplex ..\cplex
+          Xcopy /E /I D:\a\downward\cplex_temp\cplex $ENV:cplex_DIR
```

Comment:
Shouldn't this be `$ENV:GITHUB_WORKSPACE` as well?

Reply:
Don't think so. The D:\a\downward path is hardcoded.